### ISSUE TITLE
[1.12.2] Mod crashing on loading screen

### ISSUE THREAD

**Issue by RedyRT** [OreCruncher/DynamicSurroundings#711]
**Mod Version:**

DynamicSurroundings-1.12.2-3.6.2.1

**Forge Version:**

Forge 14.23.5.2855

**Link to client log:**

[crash-2021-04-12_20.27.58-client.txt](https://github.com/OreCruncher/DynamicSurroundings/files/6298673/crash-2021-04-12_20.27.58-client.txt)

**Link to crash log:**

[latest.log](https://github.com/OreCruncher/DynamicSurroundings/files/6298648/latest.log)

**Description:**

I was tried to figure it out myself this problem, only what I know, there is a problem with some biomes in the crash log.
I hope you will help me

**Comment by OreCruncher** [comment]
Do you have MixinBootstrap installed?

**Comment by RedyRT** [comment]
> 
> 
> Do you have MixinBootstrap installed?

Yes, I tried to run it with MixinBootstrap, but it just doesn't work
